.github/workflows/deploy-production-ec2.yml

```diff
@@ -0,0 +1,53 @@
+on:
+  push:
+    tags:
+      - 'v*'
+name: Deploy Production
+jobs:
+  deploy-production-1:
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install SSH Key
+        uses: shimataro/ssh-key-action@v2.3.0
+        with:
+          key: ${{ secrets.EC2_RSA_KEY }}
+          known_hosts: ${{ secrets.EC2_KNOWN_HOSTS }}
+          name: id_rsa
+      - name: SSH to Production 1
+        run: |
+          ssh ${{ secrets.PRODUCTION_HOST_1}} -T '
+          cd docker/docs-worker-pool
+          git pull origin
+          git checkout $(git describe --tags)
+          sudo docker system prune
+          sudo docker build --no-cache --tag production .
+          sudo docker stop $(sudo docker ps -a -q)
+          sudo sh rundocker.sh
```

Comment:
in a perfect world, we can run `sudo docker build --no-cache --tag production . ` in this action and then send the image to ECR - and then fetch the new image from ECR and do any stopping, pruning, etc. on the remote **instead** of building the new docker image on the remote instance

for now, this works though.